### PR TITLE
Doc 1444

### PR DESCRIPTION
**Pull request by lennessyy**

### CODE REVIEW

File: modules/appendix/pages/notations.adoc [hunk 1]
```diff
@@ -0,0 +1,130 @@
+= Documentation Notations
+:description: Overview of
```
Comment (alexthomasTG) [inline modules/appendix/pages/notations.adoc]:
```suggestion
:description: Overview of software documentation notations in GSQL
```

File: modules/appendix/pages/notations.adoc [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+RUN JOB -n last_line_num job_name
+RUN JOB -n first_line_num , last_line_num job_name
+----
+
+== Quotation Marks
+When quotation marks are shown, they are to be typed as shown (unless stated otherwise).
+A placeholder for a string value will not have quotation marks in the template code, but if a template is converted to actual code, quotation marks should be used around string values.
+
+== Choices
+The vertical bar | is used to separate the choices, when the syntax requires that the user choose one out of a set of values.
```
Comment (alexthomasTG) [inline modules/appendix/pages/notations.adoc]:
```suggestion
The vertical bar | is used to separate the choices when the syntax requires that the user choose one out of a set of values.
```

File: modules/appendix/pages/notations.adoc [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+[source,gsql]
+----
+RUN JOB job_name
+RUN JOB -n last_line_num job_name
+RUN JOB -n first_line_num , last_line_num job_name
+----
+
+== Quotation Marks
+When quotation marks are shown, they are to be typed as shown (unless stated otherwise).
+A placeholder for a string value will not have quotation marks in the template code, but if a template is converted to actual code, quotation marks should be used around string values.
```
Comment (alexthomasTG) [inline modules/appendix/pages/notations.adoc]:
```suggestion
A placeholder for a string value does not have quotation marks in the template code, but if a template is converted to actual code, quotation marks should be used around string values.
```

File: modules/appendix/pages/notations.adoc [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+----
+
+Possible actual values:
+
+[source,gsql]
+----
+LOAD "data/users.csv" TO VERTEX user VALUES ($0, $1, $2)
+----
+
+The user-defined identifiers are _edge_type_ name_ , _vertex_type_name1_, _vertex_type_name2_,      _attribute_name_ and _default_value_ . As explained in the Create Vertex section, *type* is one of the attribute data types.
```
Comment (alexthomasTG) [inline modules/appendix/pages/notations.adoc]:
```suggestion
The user-defined identifiers are _edge_type_name_ , _vertex_type_name1_, _vertex_type_name2_,      _attribute_name_ and _default_value_ . As explained in the Create Vertex section, *type* is one of the attribute data types.
```

File: modules/intro/pages/index.adoc [hunk 5]
```diff
@@ -10,132 +10,61 @@ The GSQL Language Reference is divided into two major parts:
...
 
-This guide uses conventional notation for software documentation.
-In particular, note the following:
+GSQL is the choice for fast and scalable graph operations and analytics.
+GSQL’s similarity to SQL, high-level syntax, Turing completeness, and built-in parallelism brings faster performance, faster development and the ability to describe any algorithm.
 
-=== Long lines
+=== Designed for analytics
+GSQL is designed for complex graph analytics.
+By taking advantage of a unique feature of the GSQL query language - accumulators, users can leverage TigerGraph's Massively Parallel Processing (MPP) capability without the pain and fuss traditionally associated with parallel programming.
```
Comment (alexthomasTG) [inline modules/intro/pages/index.adoc]:
```suggestion
By taking advantage of a unique feature of the GSQL query language - accumulators - users can leverage TigerGraph's Massively Parallel Processing (MPP) capability without the pain and fuss traditionally associated with parallel programming.
```

File: modules/intro/pages/index.adoc [hunk 6]
```diff
@@ -10,132 +10,61 @@ The GSQL Language Reference is divided into two major parts:
...
 
-For more convenient display, long statements in this guide may sometimes be displayed on multiple lines.
-This is for display purposes only; the actual code must be entered as a single line (unless the multi-line mode is used).
-When necessary, the examples may show a shell prompt before the start of a statement, to clearly mark where each statement begins.
 
-Example: A `SELECT` query is grammatically a single statement, so GSQL requires that it be entered as a single line.
+=== Turing-complete
+GSQL is Turing-complete with full support for imperative and procedural programming, ideal for algorithmic computation.
+The Turing completeness of the language, especially with the conventional xref:querying:control-flow-statements.adoc[control flow statements], allow users to describe any algorithm with GSQL.
```
Comment (alexthomasTG) [inline modules/intro/pages/index.adoc]:
```suggestion
The Turing-completeness of the language, especially with the conventional xref:querying:control-flow-statements.adoc[control flow statements], allows users to describe any algorithm with GSQL.
```

**Review by alexthomasTG — APPROVED** [review]

File: modules/intro/pages/index.adoc (hunk 6 above)
Comment (lennessyy) [reply modules/intro/pages/index.adoc]: https://en.wikipedia.org/wiki/Turing_completeness#:~:text=In%20colloquial%20usage%2C%20the%20terms,purpose%20computer%20or%20computer%20language.

Wikipedia doesn't seem to use the hyphen

**Pull request merged**